packages/fast-components-react-msft/src/radio/README.md

```diff
@@ -0,0 +1,8 @@
+# Radio
+An element that can be turned on and off. *Radio* buttons are almost always listed together in groups.
```

Comment:
See other comments

Reply:
going with: "Only one radio button within a group of items can be selected at the same time."